src/fragments/lib/storage/flutter/sqlite3_wasm.mdx

```diff
@@ -0,0 +1,2 @@
+Amplify uses SQLite via the drift package. This requires a custom build of SQLite when targeting web. You will need to download a sqlite3.wasm file from the releases of the sqlite3 pub package and put this file in your web/ folder.
```

Comment:
I wasn't able to find the link to download the SQLite file. Can we be more specific about the link here?

Reply:
Fixed, thanks for catching it.